### PR TITLE
Store server masterfile in consul

### PR DESCRIPTION
**Pull request by skaes**
Can someone please review?

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -82,6 +82,13 @@ namespace :redis do
   end
 end
 
+namespace :consul do
+    desc "start consul agent in development mode"
```
[the remark this answers is not in the document]
Comment (skaes) [reply Rakefile]: oops

[the remark this answers is not in the document]
Comment (skaes) [reply Rakefile]: fixed

File: README.rdoc [hunk 2]
```diff
@@ -120,9 +121,11 @@ distribution.
 
 1. Fork it
 2. Create your feature branch (`git checkout -b my-new-feature`)
-3. Commit your changes (`git commit -am 'Add some feature'`)
-4. Push to the branch (`git push origin my-new-feature`)
-5. Create new Pull Request
+3. Hack along and test your code using rake test and rake
+   cucumber. You must start consul in order to use cucumber.
```
[the remark this answers is not in the document]
Comment (skaes) [reply README.rdoc]: Actually I'd rather remove the second README and merge it's contents with the main README. I had completely forgotten about the second README.

[the remark this answers is not in the document]
Comment (boosty) [reply README.rdoc]: 👍

**Pull request merged**